src/pybind/mgr/cephadm/inventory.py

```diff
@@ -1045,6 +1045,18 @@ def get_unreachable_hosts(self) -> List[HostSpec]:
             )
         ]
 
+    def is_host_unreachable(self, hostname: str) -> bool:
+        # take hostname and return if it matches the hostname of an unreachable host
+        return hostname in [h.hostname for h in self.get_unreachable_hosts()]
+
+    def is_host_schedulable(self, hostname: str) -> bool:
```

Comment:
@adk3798 apart from the `maintenance` or `offline` status, can we include these to the `status` column? 
`Schedulable`
`Draining`
`Unreachable`
`Maintenance`
`Offline`

or probably later on for a more fine-grained host statuses?

Reply:
`Draining` maybe, but `Schedulable` and `Unreachable` are only meant to be internal info (e.g. `Unreachable` really is just an aggregation of `Maintenance` or `Offline` that's useful to check if we're deciding if we can take action on the host), and `Maintenance` and `Offline` are already part of the status as the first line in your comment stated.

Reply:
Okay makes sense. I was thinking on UI perspective. Because most of the time the Status column is empty. So when its schedulable and daemons can be deployed onto it, maybe putting a status to it will make it look good there!

Right now we are covering it with an `available` status only in the UI. If something like that comes from cephadm, maybe we could trust it more!

Reply:
yeah, for us empty status is basically the equivalent of something like `OK` or `Available`. There was one point where we were going to add in some default like that instead of leaving it blank, but for reasons I don't remember, we never did.